lib/app/service/nekoton_related/connection_service/transport_strategies/utils.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:nekoton_repository/nekoton_repository.dart';
+
+String getDefaultAccountName(WalletType walletType) => walletType.when(
```

Comment:
No, this is not correct using such method for all types of transport, because venom has really different names

Reply:
```
everWallet: () => 'Default',
walletV3: () => 'Legacy',
```

Reply:
Same for multisig

Reply:
🤟

Reply:
Thank you for the attentive review!